examples/neural_compressor/question-answering/run_qa.py

```diff
@@ -815,10 +833,13 @@ def compute_metrics(p: EvalPrediction):
 
     if optim_args.apply_quantization and optim_args.quantization_approach in {"static", "dynamic"}:
         model = trainer.model if isinstance(trainer.model, PreTrainedModel) else trainer.model._model
-        quantizer = INCQuantizer.from_pretrained(model)
+        quantizer = INCQuantizer.from_pretrained(model, eval_fn=eval_func)
         if optim_args.quantization_approach == "static":
             num_calibration_samples = min(len(train_dataset), optim_args.num_calibration_samples)
-            train_dataset = train_dataset.select(range(num_calibration_samples))
+            if training_args.use_ipex:
+                train_dataset = eval_dataset.select(range(num_calibration_samples))
```

Comment:
because Intel Neural Compressor need get example_inputs, which format is the a list of evaluate sample from the calibration dataset, the default for "static" is train_dataset, but the train_dataset key is different with eval_dataset for SQUAD.  I add a new name "calib_dataset" for it to clarify.

Reply:
So we will be able to remove it for neural-compressor v2.1 once `example_inputs` will not be mandatory anymore ?